### PR TITLE
css-library: Adding line-length tokens

### PR DESCRIPTION
**Pull request by Andrew565**
## Chromatic

https://3404-line-length-tokens--65a6e2ed2314f7b8f98609d8.chromatic.com

---
## Description
Closes [#3404](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3404)

## QA Checklist
CSS-Library only change.

## Screenshots
N/A

## Acceptance criteria
- [X] QA checklist has been completed
- [X] Screenshots have been attached that cover desktop and mobile screens

## Definition of done
- [X] Documentation has been updated, if applicable
- [X] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: packages/css-library/tokens/line-lengths.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "vads": {
+    "size": {
+      "line-length": {
+        "1": { "value": "44ex" },
+        "2": { "value": "60ex" },
+        "3": { "value": "64ex" },
+        "4": { "value": "68ex" },
+        "5": { "value": "72ex" },
+        "6": { "value": "88ex" },
+        "none": { "value": "999ex" }
```
Comment (jamigibbs) [inline packages/css-library/tokens/line-lengths.json:11]: Is this value `999ex` for `vads-size-line-length-none` taken from somewhere? Is that how USWDS is doing it? I'm looking at [our docs](https://design.va.gov/foundation/typography#typography-tokens) and it just says "none".

Comment (Andrew565) [reply packages/css-library/tokens/line-lengths.json:11]: I tried putting in "none" as the value and Style Dictionary spit back that it wasn't a number, so I chose an arbitrarily long length to "simulate" a "none" length.

Comment (Andrew565) [reply packages/css-library/tokens/line-lengths.json:11]: @humancompanion-usds Do we need to support a "none" line-length? Is making it arbitrarily long the right method to take given the limitations of Style Dictionary? I'm thinking that maybe just not having this token is sufficient.

Comment (jamigibbs) [reply packages/css-library/tokens/line-lengths.json:11]: @Andrew565 Maybe a value with a similar affect that style dictionary will accept is `unset`, `inherit`, `initial`, or `revert`:

https://css-tricks.com/inherit-initial-unset-revert/

Comment (humancompanion-usds) [reply packages/css-library/tokens/line-lengths.json:11]: Yes we should have a none token and yes "inherit" should probably be the value.

**Comment by Andrew565** [comment]
@jamigibbs Ready for re-review.

**Pull request merged**